devito/finite_differences/differentiable.py

```diff
@@ -494,6 +496,20 @@ def _diff2sympy(obj):
     return _diff2sympy(expr)[0]
 
 
+def shift_x0(shift, shape):
```

Comment:
maybe this should be moved to `devito/finite_differences/tools` ? What do you think @Leitevmd  @mloubout